src/evo/cbtx.h

```diff
@@ -23,18 +25,25 @@ class CCbTx
 public:
     static constexpr auto SPECIALTX_TYPE = TRANSACTION_COINBASE;
     static constexpr uint16_t CURRENT_VERSION = 2;
```

Comment:
should we rename `CURRENT_VERSION` to `V19_VERSION` or something similar?

Reply:
Yes we could. I agree